packages/bruno-app/src/providers/ReduxStore/slices/collections/index.js

```diff
@@ -1568,29 +1568,44 @@ export const collectionsSlice = createSlice({
         }
 
         if (type === 'request-sent') {
-          const item = collection.runnerResult.items.find((i) => i.uid === request.uid);
+          const item = collection.runnerResult.items
+            .slice()
+            .reverse()
+            .find((i) => i.uid === request.uid);
```

Comment:
Wouldn't be easier to use `.findLast((i) => i.uid === request.uid)` instead of cloning and reverting array ?

Reply:
OK, now I see @Its-treason wrote the same few hours before me :)

Reply:
Changed as suggested. No Idea why I over complicated on the first time....